### ISSUE TITLE
V6 _error on object does not prevent submitting

### ISSUE THREAD

**Issue by q42jaap** [erikras/redux-form#1107]
I have a nested object for witch I generate an `_error` field in the errors object (sync validation).

I'm using this to be able to have validation on individual fields in the object itself and for the nested object.
(let's suppose it's comparing passwords and wanting to set the error on the `passwords` field:
```js
{
  passwords: {
    password: '',
    passwordConfirm: ''
  }
}
```

I want an error on `passwords.password` *and* on `passwords`: 
```js
errors = {
  passwords: {
    _error: 'passwords do not match',
   password: 'password too weak'
  }
}
```
(this example is a bit contrived)

`_error` just works for `Field`, not only for `FieldArray`. But the `_error` does not prevent submitting.
also, `valid` gives the 'wrong' warning.

I'm using `<Field name="passwords" component={DisplayError}>` to display the error on the nested object (adding a Field passwords._error will not work because it does not display the error).

using v6.0.0-alpha.15

**Comment by q42jaap** [comment]
What I'd suggest is to add solid _error support, not only for FieldArray, but also for Field.

I'm guessing the `errors` object not being empty is not a reason not to submit (you need to have the fields registered, to support the wizard style forms).

So, if you have a field registered of type "Field", then also check for `$fieldName + '._error'`

**Comment by erikras** [comment]
I'm confused. In your example, is your field `<Field name="passwords"/>` or `<Field name="passwords.password"/>`?

**Comment by clayne11** [comment]
I'm having this issue as well. Setting the `_error` property on a field with nested values does not prevent the from being submitted.

I think the best approach is to change the shape of the error object as I propose in #1310.

**Comment by erikras** [comment]
@clayne11's fix published in [`v6.0.0-rc.5`](https://github.com/erikras/redux-form/releases/tag/v6.0.0-rc.5).
